daemon/cmd/daemon_main.go

```diff
@@ -680,17 +680,19 @@ func init() {
 	flags.MarkHidden(option.CMDRef)
 	option.BindEnv(option.CMDRef)
 
-	flags.Int(option.ToFQDNsMinTTL, 0, fmt.Sprintf("The minimum time, in seconds, to use DNS data for toFQDNs policies. (default %d when --tofqdns-enable-poller, %d otherwise)", defaults.ToFQDNsMinTTLPoller, defaults.ToFQDNsMinTTL))
```

Comment:
I'm ok removing this but since one can still choose to enable the poller this logic still applies (at https://github.com/cilium/cilium/blob/master/pkg/option/config.go#L2149-L2156). It might be best to edit this comment once that logic is removed.